docs/source/algorithms/PDFFourierTransform-v2.rst

```diff
@@ -125,6 +125,7 @@ Usage
     import numpy as np;
     xx = np.array(range(0,100))*0.1
     yy = np.exp(-(2.0 * xx)**2)
+    yy = np.delete(yy,-1) # need one less Y value than X value for histogram data
     ws = CreateWorkspace(DataX=xx, DataY=yy, UnitX='MomentumTransfer')
     Rt = PDFFourierTransform(ws, SofQType='S(Q)', PDFType='g(r)')
```

Comment:
I noticed that in this doc test example the code detects a zero in the first bin during DetermineMinIndex and moves minindex along one to "1". This is slightly confusing because the ws dataset has Y=1 in the first bin. The reason this is happening is that the code converts the input S(Q) profile to S(Q)-1 before doing the integral.

Normally S(Q) tends to "1" at high Q for the type of samples we're looking at for total scattering so I think the exp(-q^2) function that's used here is actually a better approximation to S(Q)-1. So please can you change SofQType='S(Q)-1' in the input parameters to PDFFourierTransform.

Reply:
This problem was part of my reason for wanting to set 0 as a normal value! I'm very happy to have this corrected to S(Q)-1!!!